networkx/utils/misc.py

```diff
@@ -175,3 +175,11 @@ def dict_to_numpy_array1(d,mapping=None):
         i = mapping[k1]
         a[i] = d[k1]
     return a
+
+def iterlen(iterator):
+    """ Returns the length of the iterator.
```

Comment:
Small suggestion: "Returns the number of elements in the iterator.", since iterators don't have a "length".